packages/atomic/cypress/integration/result-list/result-list-utils.ts

```diff
@@ -1,28 +1,40 @@
+import {
+  ResultDisplayDensity,
+  ResultDisplayImageSize,
+  ResultDisplayLayout,
+} from '../../../src/components/atomic-result/atomic-result-display-options';
 import {resultListComponent} from './result-list-selectors';
 
-export function withAnySectionnableResultList(assertions: () => void) {
+export function withAnySectionnableResultList(
+  assertions: (
+    display: ResultDisplayLayout,
+    imageSize: ResultDisplayImageSize,
+    density: ResultDisplayDensity
+  ) => void
+) {
   const viewports = {mobile: 1023, desktop: 1024};
   Object.entries(viewports).forEach(([viewport, width]) =>
     describe(`with a ${viewport} viewport`, () =>
-      ['list', 'grid'].forEach((display) =>
+      (['list', 'grid'] as const).forEach((display) =>
```

Comment:
Why the `as const`?

Reply:
just a short way to make those arrays typed